CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# dbt_netsuite v0.UPDATE.UPDATE
+# dbt_netsuite v0.8.1
+
+## Bugfix
+- Change the way `accounting_book_id` was being brought into the `int_netsuite2__acctxperiod_exchange_rate_map` model. Originally from `int_netsuite2__consolidated_exchange_rates`, this was resulting in duplicate records downstream in the `netsuite2__transaction_details` model due to the way it was being joined. However, now we have brought in `accounting_book_id` (accountingbook) via the `stg_netsuite2__consolidated_exchange_rates` model, so we have removed the need for `int_netsuite2__consolidated_exchange_rates` ([PR #74](https://github.com/fivetran/dbt_netsuite/pull/74))
```

Comment:
Can this be rephrased to first identify the update that is directly being applied. And then maybe in a sub bullet we can describe why this update is needed.

Reply:
updated!